strawberry/scalars.py

```diff
@@ -1,10 +1,48 @@
+import base64
 from typing import Any, Dict, NewType, Union
 
-from .custom_scalar import ScalarDefinition, ScalarWrapper
+from .custom_scalar import ScalarDefinition, ScalarWrapper, scalar
 
 
 ID = NewType("ID", str)
 
+JSON = scalar(
+    NewType("JSON", object),  # mypy doesn't like `NewType("name", Any)`
+    description="The `JSON` scalar type represents JSON values as specified by"
+    " [ECMA-404](http://www.ecma-international.org"
+    "/publications/files/ECMA-ST/ECMA-404.pdf).",
+    specified_by_url="http://www.ecma-international.org"
+    "/publications/files/ECMA-ST/ECMA-404.pdf",
+    serialize=lambda v: v,
+    parse_value=lambda v: v,
+)
```

Comment:
```suggestion
JSON = scalar(
    NewType("JSON", object),  # mypy doesn't like `NewType("name", Any)`
    description=(
        "The `JSON` scalar type represents JSON values as specified by "
        "[ECMA-404]"
        "(http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf)."
    ),
    specified_by_url=(
        "http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf"
    ),
    serialize=lambda v: v,
    parse_value=lambda v: v,
)
```

I'm a bit confused why `description` uses a Markdown (?) formatted link?